src/view/com/util/post-embeds/index.tsx

```diff
@@ -95,14 +96,9 @@ export function PostEmbeds({
       const openLightbox = (index: number) => {
         store.shell.openLightbox(new ImagesLightbox(items, index))
       }
-      const onPressIn = (index: number) => {
-        const firstImageToShow = items[index].uri
-        RNImage.prefetch(firstImageToShow)
-        items.forEach(item => {
-          if (firstImageToShow !== item.uri) {
-            // First image already prefetched above
-            RNImage.prefetch(item.uri)
-          }
+      const onPressIn = (_: number) => {
+        InteractionManager.runAfterInteractions(() => {
+          Image.prefetch(items.map(i => i.uri))
```

Comment:
Tried a few things with no luck. Only significant change to the post-embed functionality was here.

Evidently `prefetch` was blocking the UI thread, so `TouchableOpacity` was triggered and then never released/blurred. This seems to fix it.